src/ShellyU/shelly_init.cpp

```diff
@@ -48,8 +50,18 @@ void CreateComponents(std::vector<std::unique_ptr<Component>> *comps,
                       std::vector<std::unique_ptr<mgos::hap::Accessory>> *accs,
                       HAPAccessoryServerRef *svr) {
   CreateHAPSwitch(1, mgos_sys_config_get_sw1(), mgos_sys_config_get_in1(),
-                  comps, accs, svr, true /* to_pri_acc */,
+                  comps, accs, svr, false /* to_pri_acc */,
                   nullptr /* led_out */);
+
+  // Sensors
+  for (int i = 0; i < 2; i++) {
```

Comment:
any particular reason to have 2 sensors here rather than 3? you have se3 defined for ShellyU as well.

Reply:
not in particular, just a specific test-case...

Reply:
so ts3 is not used. please remove it or increase number of created sensors to 3.